cvat-core/src/frames.ts

```diff
@@ -89,11 +89,8 @@ export class FramesMetaData {
     public includedFrames: number[] | null;
     public frameFilter: string;
     public frames: {
-        width: number;
-        height: number;
-        name: string;
-        related_files: number;
-    }[];
+        [index: number]: SerializedFrameMetaData;
```

Comment:
Changing interface of public member in that way is a dangerous update 
I see this field may be accessed in some places, including enterprise code. I suspect all this code must be updated correspondingly. 

Do we have any other options?